src/inject/GPMInject/interface/customUI.js

```diff
@@ -71,14 +90,29 @@ window.wait(() => {
     return false;
   });
   document.querySelectorAll('.nav-section.material')[0].insertBefore(dSettings, document.querySelectorAll('.nav-section.material > a')[2]); // eslint-disable-line
-
-  // Back button
-  const back = document.createElement('paper-icon-button');
-  back.addEventListener('click', () => history.back());
-  back.setAttribute('icon', 'arrow-back');
-  back.setAttribute('id', 'backButton');
-  back.setAttribute('class', 'x-scope paper-icon-button-0');
-  document.querySelector('#material-one-middle > sj-search-box').insertBefore(back, null);
+}
+
+/** Create the back button. */
+function installBackButton() {
+  const listenNowURL = 'https://play.google.com/music/listen#/now';
+
+  const backBtn = document.createElement('paper-icon-button');
+  backBtn.setAttribute('icon', 'arrow-back');
+  backBtn.setAttribute('id', 'backButton');
+  backBtn.setAttribute('class', 'x-scope paper-icon-button-0');
+  document.querySelector('#material-one-middle > sj-search-box').insertBefore(backBtn, null);
+
+  backBtn.addEventListener('click', () => {
+    const oldUrl = location.href;
```

Comment:
Considering I wrote that on my phone while walking home I am quite impressed it just worked :+1: